assets/src/domain/eventEditor/ui/datetimes/datesList/filterBar/useDatesListFilterState/index.test.ts

```diff
@@ -205,36 +205,6 @@ describe('useDatesListFilterState', () => {
 		expect(datesOrder).toEqual(expectedDatesOrder);
 	});
 
-	test('should update processedDates to reflect changes made by invoking setShowDates with activeOnly filter', () => {
```

Comment:
At some point I will get rid of this file and apply tests only for specific predicates.